packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -18,10 +18,25 @@ open class ExpoAppDelegate: UIResponder, UIApplicationDelegate {
   @objc
   public let reactDelegate = ExpoReactDelegate(handlers: reactDelegateHandlers)
 
+    @objc
+    open override func buildMenu(with builder: UIMenuBuilder) {
+        let parsedSubscribers = subscribers.filter {
+            $0.responds(to: #selector(buildMenu(with:)))
+        }
+
+        if parsedSubscribers.isEmpty {
```

Comment:
It doesn't seem to be needed, you only use `forEach` after that so it already does the same thing when the array is empty 😅